examples/k8s/value_tso_HA.yaml

```diff
@@ -0,0 +1,449 @@
+# Default values for byconity.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+# please replace  storageClassName
+nameOverride: ""
+fullnameOverride: ""
+
+byconity:
+  image: byconity/byconity:0.1.0-GA
+  imagePullPolicy: IfNotPresent
+  hdfs_address: hdfs://byconity-hdfs-namenodes:8020 # can using your own hdfs
+  #  use_existing_fdb: true
+  #fdb_cluster_file: byconity_fdb:Is0hBgl6iICdHuspBmhAODmD5WISXKzI@192.168.224.150:4501,192.168.226.83:4501,192.168.228.152:4501
+  ports:
+    tcp: 9000
+    http: 8123
+    rpc: 8124
+    tcpSecure: 9100
+    https: 9123
+    exchange: 9410
+    exchangeStatus: 9510
+
+  usersOverwrite:
+    users:
+      default:
+        password: ""
+      probe:
+        password: probe
+
+  server:
+    replicas: 2
+    image: ""
+    podAnnotations: { }
+    resources: { }
+    hostNetwork: false
+    nodeSelector: { }
+    tolerations: [ ]
+    affinity: { }
+    imagePullSecrets: [ ]
+    securityContext: { }
+    livenessProbe:
+      exec:
+        command: ["/opt/byconity/scripts/lifecycle/liveness"]
+      failureThreshold: 6
+      initialDelaySeconds: 5
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 20
+    readinessProbe:
+      exec:
+        command: ["/opt/byconity/scripts/lifecycle/readiness"]
+      failureThreshold: 5
+      initialDelaySeconds: 10
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 10
+    storage:
+      localDisk:
+        pvcSpec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 30Gi
+          storageClassName: openebs-hostpath #replace to your storageClassName
+      log:
+        pvcSpec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 20Gi
+          storageClassName: openebs-hostpath #replace to your storageClassName
+    configOverwrite:
+      logger:
+        level: trace
+      disk_cache_strategies:
+        simple:
+          lru_max_size: 429496729600 # 400Gi
+      # timezone: Etc/UTC
+    hdfs3Config:
+      dfs.client.metrics.enable: true
+      dfs.client.metrics.use_domain_socket: true
+      dfs.default.blocksize: "268435456"
+      dfs.read.hedge_read.interval_ms: 120
+      input.connect.timeout: 10000
+      input.read.max.retry: 3
+      input.read.timeout: 6000
+      input.write.timeout: 10000
+      output.close.timeout: 256000
+      output.connect.timeout: 10000
+      output.default.write.retry: 3
+      output.read.timeout: 120000
+      output.write.timeout: 256000
+      rpc.client.connect.retry: 3
+      rpc.client.connect.timeout: 10000
+      rpc.client.read.timeout: 20000
+      rpc.client.timeout: 20000
+      rpc.client.write.timeout: 20000
+
+  tso:
+    replicas: 3
+    image: ""
+    podAnnotations: { }
+    resources:
+      limits:
+        cpu: "4"
```

Comment:
seems the request for cpu is too high, maybe 3 is enough

Reply:
i remove this part.